### DISCUSSION TITLE
Calculation period change

### DISCUSSION THREAD

**Discussion by lyselott** [Universite-Gustave-Eiffel/NoiseModelling#605] [Q&A] [answered]
Hi,
It is possible to change the calculation periods within any of the WPS blocks or directly in Noise_level_from_source?

For example, in Italy the day (06-22) and night (22-06) are different from those defined in END.

Also, I would like to know if it is possible to reduce the periods to less than an hour to do the calculation in minutes. I know that is possible with MATSIM_ROADS_STATS, but I would like to be able to do it directly without MATsim.

Best,
Alexandra

**Comment by pierromond** [comment, marked as answer]
Here you can change the duration of each period and the ponderation : https://github.com/Universite-Gustave-Eiffel/NoiseModelling/blob/5e137000d5e39fb0e42e9802c0b263c5a9d570b4/noisemodelling-jdbc/src/main/java/org/noise_planet/noisemodelling/jdbc/LDENComputeRaysOut.java#L311

Unluckly, it is not so easy to do, we should improve it !

About the duration, in fact you can to the time step that you want, this is transparent for NoiseModelling. You can also store the attenuation matrix, and then run a lot of emission calculation (every minutes for example) and then couple it with the attenuation. If you look inside the MATSim script, this is what is does ! Probably we should write a routine with documentation also to do that !
